press-release-faq.md

```diff
@@ -134,7 +138,9 @@ In keeping with our goal to be minimally different from vanilla (p)react, we hav
 
 #### How do cross-components function calls work?
 
-> @andy (very high level summary then link to architecture doc)
+Containers maintain a set of callbacks, defined within the container, which are available to be "invoked" across container boundaries. This includes functions passed via `props` and function arguments passed to `props` functions. When an external container needs to invoke one of these callbacks, the external container requests the outer application to send a message to the target container identifying the method and arguments.
+
+See [architecture.md](architecture.md#component-callbacks) for more details.
 
 #### What are the downsides of using iframes?
```

Comment:
This is a bit broad - what are we trying to answer here?

Are we contrasting iframes to other solutions listed in this document? Enumerating the constraints this imposes on BWE?

Reply:
I was going for enumerating the constraints it imposes on BWE. If you don't feel its a valid FAQ entry it can be dropped